README.md

```
@@ -71,105 +47,190 @@ let cropViewController = Mantis.cropViewController(image: <Your Image>, config:
+
+- iOS 15.0+
+- macOS 12.0+ (Mac Catalyst)
+- Xcode 13.0+ (Xcode 15+ recommended — enables the iOS 17 Observation-based fine-grained tracking in `CropSession`)
+
+## Installation
+
+### Swift Package Manager (recommended)
+
+In Xcode: **File → Add Package Dependencies…** and enter:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)